src/main/java/com/example/demo/repository/EmailRepo.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.repository;
+
+import com.example.demo.consoleInfo.Email;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface EmailRepo {
+
+    void save(List<Email> email) throws IOException;
+
+    List<Email> findAllEmails() throws IOException;
+
+    List<Email> showEmailsForSending() throws IOException;
```

Comment:
Let's rename it to findEmailsForSending to comply with repository conventions. This component is a storage and is not responsible for displaying information. That's why term "show" doesn't fit here